### ISSUE TITLE
Relays are down

### ISSUE THREAD

**Issue by sveken** [Yellow-Dog-Man/Resonite-Issues#853]
### Describe the bug?

The only relay currently online is the EU one, this results in say myself in Australia getting routed via the EU before reaching my destination resulting in deysnc. 

### To Reproduce

Use -ForceRelay or be on a connection requiring a relay. 

### Expected behavior

More relays  are available like the Australian one 

### Resonite Version Number

2023.11.28

### What Platforms does this occur on?

Windows

### Log Files

https://api.resonite.com/networknodes

### Reporters

sveken
bredo

**Comment by JackTheFoxOtter** [comment]
Interestingly, the status page doesn't seem to reflect that outage (https://status.yellowdogman.com/)...

**Comment by sveken** [comment]
Morning @shiftyscales just woke up myself.

As for https://api.resonite.com/networknodes It only lists a single LNL_relay instead of the multiples that should be available still.

To test there is still issues i have connected to a headless that is in Sydney. On the relay this would result in a reported ping less then 2ms as it was basically in the same location as the AU relay. 
Right now, i am getting a ping of 144ms to this same headless, which i belive is due to being relayed through the only available LNL-relay mentioned on the networksnodes page. 

Looking up the geo location of the ip 95.217.33.19  shows finland and a 144ms (so 290ish pretty much round trip as its halved) ping time lines up with that location well.

**Comment by shadowpanther** [comment]
I wonder whether someone could also check why LNL Relays don't work for me at all - #44 
It severely limits my connection possibilities as LNL NAT also doesn't work for me because of my network configuration and SNS requires both me and the host to run Steam.

Sorry in advance for writing in this issue, but seeing newer issues get triaged and older ones not touched by anyone at all is a bit frustrating.

**Comment by ProbablePrime** [comment]
Non-EU relays are now up, https://api.resonite.com/networknodes now also lists the names for the JP and AUs Relay

Special thanks to @Sveken for making me aware AND filing a GH report.
